examples/integrations/README.md

```diff
@@ -35,6 +39,23 @@ Integrations are created for an API using the `integration:create` command. The
 |apiId|string|no|AWS ID of the API to update. Empty value will create a new API in AWS. This property is optional.|
 |enabled|boolean|no|Enables the integration, if set to `false` the integration will be saved but will not execute. Default value is `true`.|
 
+## Amazon API Gateway Lambda Integration
+|Property|Type|Required|Description|
+|-|-|-|-|
+|name|string|yes|Display name of the integration. Must be unique among all integrations configured for the given API version.|
+|configType|string|yes|"AMAZON_API_GATEWAY_LAMBDA" is used to create an Amazon API Gateway Lambda integration.|
+|region|string|yes|AWS region where the API will be published. See [*Property: `region`*](#property-region) for options.|
+|runtimeLanguage|string|no|The target runtime language to generate lambda functions. Options are: `nodejs12.x`, `nodejs10.x`, `python3.8`, `python3.7`, `python3.6`, `python2.7`. Default value is `nodejs12.x`.|
+|lambdaRole|string|no|Execution role for creating non-existing Lambda Functions. If specified, missing lambda functions will be created using this role. Must be in the `arn:aws:iam::{ID}:role/{NAME}` format.|
+|accessKey|string|yes|AWS access key.|
+|secretKey|string|yes|AWS secret key.|
+|publishMode|string|no|How to update an existing API in AWS. Options are: `merge`, `overwrite`. Default value is `merge`.|
+|basePathMode|string|no|How to handle the API's basePath value. Refer to [AWS documentation](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-import-api-basePath.html) for details. Options are `ignore`, `prepend`, `split`. Default value is `ignore`.|
+|updateDefinition|boolean|no|Whether to update the API definition with Amazon-specific extensions and compatibility modifications. Default value is `false`.|
+|deploymentMode|string|no|Should be `on save`. The value `never` means the integration is disabled. Default value is `on save`.|
```

Comment:
```suggestion
|deploymentMode|string|no|Should be `"on save"`. The value `"never"` means the integration is disabled. Default value is `"on save"`.|
```

Reply:
Just on this one, do you think all the enum values should have the `""` or is it just because there's a space in the value.

Reply:
I thought it would be clearer using quotation marks due to the space, I don't think it's necessary in general.